siff-dr-router/siff-header.go

```diff
@@ -0,0 +1,328 @@
+package main
```

Comment:
We kind of have to because docker.  I'm working on a  way around this which is basically making all of the containers use the same image but with different command line arguments